src/altinn-app-frontend/package.json

```diff
@@ -28,6 +28,7 @@
     "ajv": "^8.11.0",
     "ajv-formats": "^2.1.1",
     "axios": "^0.27.2",
+    "css-mediaquery": "^0.1.2",
```

Comment:
Do we need this dependency? We have other tests that set screen size to simulate mobile screens, f.ex https://github.com/Altinn/app-frontend-react/blob/main/src/altinn-app-frontend/src/components/base/DatepickerComponent.test.tsx#L46-L61
This could be extracted to a generic testutil f.ex (not saying you need to do this as part of this PR :) ) but I would like to avoid adding a dependency that hasnt seen updates in 8 years :)